js/metabox-draggable.js

```diff
@@ -0,0 +1,109 @@
+jQuery(document).ready(function($){
+	/**
+	 * Initialize sortable
+	 */
+	$("#dragdrop-sortable").sortable({
+		handle: ".dragdrop-sortable-item-header",
+		placeholder: "dragdrop-highlight"
+	});
+
+    /**
+     * Update numeric order
+     */
+    function dragdropChnageItemOrder(){
+    	var allDragdropItems = $('.dragdrop-sortable-content li.single-sortable-item');
+    	$.each(allDragdropItems, function(index, value){
+            var childItem = allDragdropItems.eq(index);
+            childItem.find('input.dragdrop-set-order').val(index);
+    		childItem.find('span.dragdrop-item-count').html(index+1);
```

Comment:
Not a big deal at all, just seeing this small spacing inconsistency. I have [this ESLint extension](https://marketplace.visualstudio.com/items?itemName=dbaeumer.vscode-eslint) for VSCode that shows issues and errors inline, which is super nice and highly recommended! It also flags things like whether double or just single quotes are needed (which is probably the case in the `sortupdate` function below).

I also know we don't have any ESLint configuration set up on this repo 🙃, but the extension can be configured with a global fallback, which is also very cool.

Reply:
Ah, the ugly results of copied and pasted code! Thanks for your sharp eyes - I've updated both the typo and the spacing. I have that ESLint extension but I must not have something set quite right as it isn't flagging quotes.